frontend/javascripts/oxalis/merger_mode.js

```diff
@@ -97,56 +96,81 @@ async function createNodeOverwrite(store, call, action, mergerModeState: MergerM
   if (!segmentId) {
     api.utils.showToast("warning", messages["tracing.merger_mode_node_outside_segment"]);
   } else {
-    call(action);
+    await call(action);
+    // We center the create cell manually, as we implicit override the action that centers the new cell automatically.
```

Comment:
```suggestion
    // We center the create cell manually, as we implicitly override the action that centers the new cell automatically.
```

Reply:
Interesting. I would have thought, that `call(action);` still dispatches the original action which is why no behavior is lost (i.e., the centering should still happen). But I'm probably wrong?

Reply:
I just tried out why I added the explicit call to center the active node.

The reason is that somehow the centering messes up and centers the last created node. By adding the centerActiveNode` call this issue is fixed.

As this was easy to fix, I did not debug why actually the last node is centered and the one that just got added.

Do you think it is fine to keep it this way, @philippotto? Or shall I look why the last cell is centered instead of the one that just got added?

Reply:
I think, it's fine as is. But maybe extend the comment with the information you just stated here :)